x/crosschain/types/keys.go

```diff
@@ -23,7 +23,7 @@ const (
 	// we keep the current value for backward compatibility
 	MemStoreKey = "mem_metacore"
 
-	ProtocolFee = 2000000000000000000
+	ProtocolFee = 250000000000000000
```

Comment:
should this be 
500000000000000000 , according to the description

Reply:
okay I see there are a few comments around this already. Maybe update the description based on the final decision